src/pages/CollectionPage/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react"
+import { Container, Row, Col, Card } from 'react-bootstrap';
+import { Collection } from "@imtbl/core-sdk";
+import HeaderBar from "../../components/HeaderBar/HeaderBar"
+import { getCollections } from "../../api/collections-api";
+import { Link } from "react-router-dom";
+import placeholderImg from '../../asset/placehoderImg.jpg';
+
+const CollectionsPage = () => {
+    const [result, setResult] = useState<Collection[]>([])
+
+    useEffect(() => {
+        getCollections().then((response) => {
+            setResult(response.result)
+        })
+    }, [])
+
+    return (
+        <>
+            <HeaderBar />
+            <Container>
+                <Row>
+                    {!!result.length && result.map((item: Collection, index) => {
+                        return (
+                            <Col xs={12} sm={6} md={4} key={index}>
+                                <Link to={`/collection/` + item.address} style={{ textDecoration: 'none', color: 'black' }}>
```

Comment:
might be a future refactoring task but if we're gonna do this for all Links consider making a custom component